package.json

```diff
@@ -7,8 +7,8 @@
     "doc": "docs"
   },
   "scripts": {
-    "sass": "sass candis/app/client/styles/App.scss:candis/app/assets/css/styles.min.css --sourcemap=none --style compressed",
-    "sass.watch": "sass --watch candis/app/client/styles/App.scss:candis/app/assets/css/styles.min.css --sourcemap=none --style compressed",
+    "sass": "node-sass candis/app/client/styles/App.scss:candis/app/assets/css/styles.min.css --sourcemap=none --style compressed",
```

Comment:
with reference to https://github.com/sass/node-sass#command-line-interface, `--style` should be replaced with `--output-style`, and there should be no `:` in between input and output file path.